arangod/GeoIndex/Index.cpp

```diff
@@ -49,8 +50,10 @@ namespace geo_index {
 
 Index::Index(VPackSlice const& info,
              std::vector<std::vector<basics::AttributeName>> const& fields)
-    : _variant(Variant::NONE) {
+    : _variant(Variant::NONE), _legacyPolygons(false) {
```

Comment:
does this initialization have to take place in the initializer list as we're doing something with it in the constructor block?